config/config.js

```diff
@@ -1,3 +1,5 @@
 var db = 'mongodb://localhost/car-pooling-database';
+var secret = '51C0BDFF3A6AF62168844A5E43E2789B4D5AFC96B1354F227AA46F0F6FC604D4';
```

Comment:
Config values don't have to be stored in the repo, because of security reasons. You can ignore the file and keep a sample file of it in the repo.